cmd/minishift/cmd/start_preflight.go

```diff
@@ -534,8 +534,8 @@ func validateOpenshiftVersion() bool {
 	return true
 }
 
-// checkOcTagOnGithub return true if specified version of OpenShift released.
-func checkOcTagOnGithub() bool {
+// checkOpenShiftRelease return true if specified version of OpenShift released
+func checkOpenShiftRelease() bool {
 	client := github.Client()
 	_, _, err := client.Repositories.GetReleaseByTag("openshift", "origin", viper.GetString(configCmd.OpenshiftVersion.Name))
```

Comment:
@praveenkumar  any specific reason we did not use `GetRelease` in place of  `GetReleaseByTag`. Techincally we need to check the release not the tag in Origin github repository?